### DISCUSSION TITLE
Help with the weather info

### DISCUSSION THREAD

**Discussion by oneeyedreaper** [lptstr/winfetch#195] [Q&A] [answered]
The displayed weather info in winfetch is incorrect. Can someone please help me with it? I'm not sure what I'm doing wrong.

**Comment by rashil2000** [comment]
What error/message are you getting?

**Reply by parasasanani** [reply]
![Screenshot](https://github.com/lptstr/winfetch/assets/92646847/02b23fe5-3d3a-4517-b19c-2d1f93a5ce88)
This is what the weather winfetch is displaying and 
![image](https://github.com/lptstr/winfetch/assets/92646847/6df208c0-f65e-4926-89b3-35b4bffff850)
this is what it's supposed to be.

So is there something I messed up during install and setup to cause this?
If so, how do I fix this?

**Reply by parasasanani** [reply]
I think I may have figured out what's causing this.
wttr.in, when visited without any location parameters, returns the weather info based on the geolocation of my public address. In my particular case, the ISP I'm using hasn't configured the locations very well and set them to static in one place. This being the case, can I do something about this? Like, edit the winfetch.ps1 file to modify the endpoint for fetching the data?

**Reply by rashil2000** [reply, marked as answer]
In th config file, you can disable the built-in weather segment, and use a custom segment to fetch weather using a proper method - see https://github.com/lptstr/winfetch/wiki/Configuration#custom-info-segments

**Reply by oneeyedreaper** [reply]
Thank you so much for your help!!
